### ISSUE TITLE
Where is the changelog and version control of this package?

### ISSUE THREAD

**Issue by ghost** [KjellConnelly/react-native-rate#91]
How do I know when a version has a breaking change or not or what has changed from one version to another? Is there a place where we have a track of these things?

**Comment by pke** [comment]
I have to agree. Unless releases, changelog is present this can not be used in production apps.
Until then I'll give https://github.com/MinaSamir11/react-native-in-app-review a try.

Also: enabling and monitoring github "discussions" feature for this repo might help to reduce clutter in issues.
